README.md

```diff
@@ -56,6 +57,19 @@ us](mailto:edgeml@microsoft.com).
 to this
 [project](https://www.microsoft.com/en-us/research/project/resource-efficient-ml-for-the-edge-and-endpoint-iot-devices/).
 
+If you use the EdgeML library in your projects or publications, please use the
+following BibTeX to cite us. Note: Author ordering is based on last names.
```

Comment:
@harsha-simhadri , this will be completely your call. As of now, we have the repo citation to be last name basis. You make the final decision on this citation. We feel that repos should have bibtex entries as Extreme Classification repo has. I have seen at least 10 papers using EdgeML codebases.